lib/bashly/views/command/run.erb

```diff
@@ -1,15 +1,15 @@
 # :command.run
 run() {
-  declare -A args
-  declare -a other_args
-  declare -a input
+  declare -A args=()
+  declare -a other_args=()
+  declare -a input=()
   normalize_input "$@"
```

Comment:
I was able to get rid of the `[[ -v arg ]]` test and the unbound error they solved, by assigning an empty array.